onebusaway-api-webapp/src/main/resources/data-sources.xml

```diff
@@ -86,4 +86,12 @@
             </props>
         </property>
     </bean>
+    <bean id="customRouteSort" class="org.onebusaway.transit_data.model.RouteSort">
+        <constructor-arg name="agencySortConfiguration" >
+            <map>
+                <entry key="MTASBWY" value="1,2,3,4,5,6,7,A,C,E,B,D,F,M,G,L,N,Q,R,W,J,Z,Sf,S,Sr,SIR"/>
```

Comment:
Let's not commit this here, as it will apply to all deployments.  This fragment belongs in dev-ops, I'll show you where

Reply:
Sure.